DependencyInjection/SonataBlockExtension.php

```diff
@@ -72,7 +72,9 @@ public function load(array $configs, ContainerBuilder $container)
         $this->configureProfiler($container, $config);
         $this->configureException($container, $config);
         $this->configureMenus($container, $config);
-        $this->configureClassesToCompile();
+        if (\PHP_VERSION_ID < 70000) {
```

Comment:
In this case, go for it.

It would be a great Fixer idea for php-cs-fixer BTW...